code-of-conduct/index.md

```diff
@@ -12,4 +12,55 @@ image:
+
+It is based on, and heavily inspired by a reading of the [Python community code of conduct](https://www.python.org/psf/codeofconduct/), the [Apache foundation code of conduct](http://www.apache.org/foundation/policies/conduct.html), and the [Debian code of conduct](https://www.debian.org/code_of_conduct)
+
+
+## The code of conduct for the NIPY community
+
+The Neuroimaging in Python (NIPY) community is made up of members with a diverse set of skills, personalities, background, and experiences. It is through these differences that our community will experience great successes and continued growth. Decisions we make affect users, colleagues, and through scientific results, the general public. We take those consequences seriously when making decisions. When you are working with members of the community, we encourage you to follow these guidelines, which help steer our interactions and help keep NIPY a positive, successful, and growing community.
+
+A member of the NIPY community is:
+
+### Open
+
+Members of the community are open to collaboration. Be it on the reuse of data, on the implementation of methods, on finding technical solutions, establishing best practices, and otherwise. We are accepting of all who wish to take part in our activities, fostering an environment where anyone can participate and everyone can make a difference.
+
+#### Be collaborative!
+
+Our work will be used by other people, and in turn we will depend on the work of others. When we make something for the benefit of others, we are willing to explain to others how it works, so that they can build on the work to make it even better. We are willing to provide constructive criticism on the work of others and accept criticism of our own work, as the experiences and skill sets of other members contribute to the whole of our efforts.
+
+#### Be inquisitive!
+
+Nobody knows everything! Asking questions early avoids many problems later, so questions are encouraged, though they may be directed to the appropriate forum. Those who are asked should be responsive and helpful, within the context of our shared goal of improving neuroimaging practice.
+
+### Considerate
+
+Members of the community are considerate of their peers. We are thoughtful when addressing the efforts of others, keeping in mind that often-times the labor was completed simply for the good of the community. We are attentive in our communications, whether in person or online, and we are tactful when approaching differing views.
+
+#### Be careful in the words that you choose:
+
+We value professionalism and courtesy in all our interactions, and take responsibility for our own speech. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary behaviour are not acceptable. This includes, but is not limited to:
```

Comment:
I have a fairly intense dislike of the the word 'professionalism'.  I don't think open-source coders should or do try and be 'professional'. 

'Includes but is not limited to' sounds specifically legal, and therefore rather threatening and unfriendly.

How about 'Please do not insult or put down others in the conversation.  Please be very careful to avoid:'